code/drasil-metadb/package.yaml

```diff
@@ -0,0 +1,33 @@
+name:                drasil-metadb
+version:             0.1.1.0
+author:              "Dan Szymczak, Steven Palmer, Jacques Carette, Spencer Smith"
```

Comment:
Remove Dan and Steven from this author list, since it is no longer true. Do feel free to add yourself.